docs/release.md

```diff
@@ -1,43 +1,65 @@
+A new TRG, or a change to an existing one is proposed by raising a PR. This can be done by any contributor or committer.
+The PR is raised against the `main` branch
+of [eclipse-tractusx/eclipse-tractusx.github.io](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/).
+Changes can be done by directly updating the existing Markdown files. New TRGs should already be put to a matching
+section. In case a new TRG does not fit do an existing section, a new section can be proposed or asked for by reviewers.
+
+As soon as a proposal is ready for committer review, it has to be announced
+via [Tractus-X mailing list](https://accounts.eclipse.org/mailing-list/tractusx-dev). Additionally, the committer group
+is mentioned directly on the PR by tagging `@eclipse-tractusx/automotive-tractusx-committers`.
+
+### Review
 
-This TRG process is open for all and has the main goal of providing reasonable defaults for workflows and requirements on being able to easily contribute, maintain and run products/components across Eclipse Tractus-X.
+Feedback to announced proposals is welcome by everybody. It should be provided by using the review functionality of
+GitHub. Reviews are especially needed from Tractus-X committers, since they are the ones who are accepting or dropping
+proposals.
 
-### Create new draft
+During review, the following things could be paid special attention to:
 
-- Create a draft version under release/trg-0
-  - Reserve the TRG number already
-- Create a pull request for merging the draft version (IF it helps to visualize / present the draft on the website) and get an approval
-- Announce the new Draft TRG on developer mailinglist
+- Does the TRG focus on a single topic? If not, consider splitting to multiple TRGs
+- Is it clear, how to comply with the TRG, or is there a lot of room for interpretation?
+- Would it be a lot of effort to comply with the TRG. Consider introducing a "mandatory from" date.
 
-#### Release fast or slow
+### Accept / Drop
 
-If you now have a small change, like a typo or updating description you do a fast release. If you create a new and big TRG (which takes time to do) you do a slow release.
+A new TRG, or changes to an existing TRG are __accepted__, if all the following is true:
 
-- **Fast**
-  - Create new Pull Request for moving the draft version to its final location
-  - Address all issues in the PR and merge
-  - Announce the new TRG on the developer mailinglist
-- **Slow**
-  - Create new Pull Request for moving the draft version as a PRERELEASE version in its final location
-  - Add a hint on when the TRG becomes mandatory
-  - Address all issues in the PR and merge
-  - Announce the new TRG on the developer mailinglist
-  - When the TRG mandatory date is due, remove the hint and move it from PRERELEASE to normal (as all other TRGs)
+- at least 7 committers did approve the change
+- the proposal PR is open for at least 2 weeks, to give everyone a fair amount of time to provide feedback or request
```

Comment:
The case which interests me is when the TRG PR doesn't get much attention from committers and doesn't get enouch approvals within 2 weeks, as per listed requirements does that get dropped? I think it might be a useful to keep traction, reminder of open TRG PRs in the Committers meeting.

Reply:
Yes that is definitely an interesting case and unfortunately the most common one, at least on the recent TRGs. 
I was not yet sure, how to state that in our process. 
You could say, that dropping is actually the right thing to do, since there does not seem to be a lot of interest across the committer group to align on such a topic. 
We could also think about "reminders", that it is a committer duty to vote on project rules, just like it is for committer elections. Just not sure, what we should do, if colleagues are constantly skipping the vote.

Reply:
As shortly discussed in our sync today:

I think it should be feasable for the project leads to still allow a TRG if there is no Veto. It should also be later on the responsible of the project lead to inform/warn committers that they have a duty and remove them later on if they don't react/act on TRGs or other things in the community.

Nonetheless if there is no real veto a TRG with small activty should be allowed as there are often enough approaches which need to be aligned

Reply:
I would like to hear more opinions on that. From my point of view, the rules are still made and discussed by the committer group. Low activity should result in a reminder to the committer group of their duties instead of "just waving through" as PL. 
Taking this to the committer office hour